src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,173 @@
+
+Dart's core libraries contain classes most commonly used by developers.
+Classes that are still useful, but not as universally used,
+are placed in packages outside of the SDK.
+Separating out the less commonly used classes ensures that your app
+isn't bloated by features that you don't need.
+For example, the
+[dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)
+library contains the Queue, HashMap, and HashSet classes. The
+[collection package](https://www.dartdocs.org/documentation/collection/latest/collection/collection-library.html)
+contains more specialized classes, such as CanonicalizedMap, MapKeySet, PriorityQueue,
+and support for equality testing.
+
+If you don't find the functionality you need on api.dartlang.org, check pub.dartlang.
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+Libraries and packages aren't interchangeable terms. Both libraries and
+applications are distributed as packages. A library can be part of a package,
+but a package is not a library.
+For more information on what constitutes a library package, see
+[Create Library Packages](/guides/libraries/create-library-packages).
+</aside>
+
+Some of the libraries in the SDK have corresponding packages with additional
+functionality. The following table includes some that you should know about:
+
+| **SDK Library** | **Package** |
+| [dart:async](https://api.dartlang.org/stable/dart-async/dart-async-library.html)<br>Support for asynchronous programming with classes such as Future and Stream. | [async](https://www.dartdocs.org/documentation/async/latest/)<br>Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html). |
+| [dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)<br>Classes and utilities that supplement the collection support in dart:core. | [collection](https://www.dartdocs.org/documentation/collection/latest)<br>Utility functions and classes to make working with collections easier. For more information, see [Unboxing Packages: collection](http://news.dartlang.org/2016/01/unboxing-packages-collection.html). |
+| [dart:convert](https://api.dartlang.org/stable/dart-convert/dart-convert-library.html)<br>Encoders and decoders for converting between different data representations, including JSON and UTF-8.| [convert](https://www.dartdocs.org/documentation/convert/latest/)<br>This package contains encoders and decoders for converting between different data representations.|
+| [dart:core](https://api.dartlang.org/stable/dart-core/dart-core-library.html)<br>Built-in types, collections, and other core functionality for every Dart program. | N/A |
```

Comment:
We probably don't need to mention dart:core here at all. It's not like you're importing it.